esda/moran.py

```diff
@@ -505,6 +508,35 @@ def Moran_BV_matrix(variables, w, permutations=0, varnames=None):
 
 
     """
+    try:
+        # check if pandas is installed
+        import pandas
+        if isinstance(variables, pandas.DataFrame):
+            # if yes use variables as df and convert to numpy_array
+            varnames = pandas.Index.tolist(variables.columns)
+            variables_n = []
+            for var in varnames:
+                variables_n.append(variables[str(var)].values)
+        else:
+            raise ImportError('install pandas in order to pass /n'
```

Comment:
Won't this be raised if `variables` is not an instance of `DataFrame`? So, if it's an array, will it raise the error? As I understand the documentation, it shouldn't, right?